tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php

```diff
@@ -30,11 +30,11 @@ final class ClassDefinitionFixerTest extends AbstractFixerTestCase
     public function testConfigureDefaultToFalse(): void
     {
         $defaultConfig = [
+            'inline_constructor_arguments' => true,
             'multi_line_extends_each_single_line' => false,
             'single_item_single_line' => false,
             'single_line' => false,
             'space_before_parenthesis' => false,
-            'inline_constructor_arguments' => true,
```

Comment:
@keradus @SpacePossum 

As an alternative to this change, I could extract a `FixerOptionSorter` and use it in both the `FixerConfigurationResolver` and `ClassDefinitionFixerTest::assertSame()`.

What do you think?

Reply:
I think we either have a low-level test that options are always sorted,
or we have a sorter mechanism that is used on that low-level (ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7345#discussion_r1349563388 )

Reply:
Having a test that checks all options for all fixers seems the way to go for me. Than there is no need to test order of the options @ runtime. (likely in `FixerConfigurationResolverTest.php`)

```
There are also other options, e.g.

    max_line_breaks
    min_line_breaks

where special ordering could make sense.
```

:+1:   that could be added to the tests, I think I a saw an option called `default` somewhere as well, maybe it might make sense to have that one as first as well for example. I just thinking out loud here

Reply:
@keradus @SpacePossum 

Not sure I understand - would you like me to to make any changes here? If so, which? Thank you!